pandas/core/generic.py

```diff
@@ -1940,6 +1940,84 @@ def tail(self, n=5):
             return self
         return self.iloc[-n:]
 
+  
+    def sample(self, n = None, frac = None , replace = False, weights = None, seed = None):
+        """
+        Returns a sample of rows from object. 
+    
+        Note that .sample() (with no arguments) will return five random rows.
+    
+        Parameters
+        ----------
+            n: Number of rows to return. Cannot be used with frac.
```

Comment:
please take another look at the numpy docstring standard:
https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt

In particular, you are indenting too much here and mixing up the lines for types/descriptions.